charts/immich/values.yaml

```diff
@@ -8,207 +8,119 @@
 image:
   repository: ghcr.io/immich-app/immich-server
   tag: v1.135.3
+  # mlFlavor: "cuda"
 
 immich:
+  # env:
+  #   REDIS_HOSTNAME: 'immich-redis.app-gallery.svc.cluster.local'
```

Comment:
is there no way to implicitly figure these out? im unfamiliar with this redis setup

Reply:
right now its defaulting to the values for db charts:

https://github.com/Gentoli/immich-charts/blob/d06d13da51c344d2ae3efabdcacbd28aebac390c/charts/immich/templates/common.yaml#L120-L123

We could add some logic to figure out these based on what the user enables, which can also add as a check for the user to not enable both. The only issue I had was `nameOverride` didn't work in `rawResources`, but doable if we only consider the default name.

Reply:
FYI the operator manages a bunch of resources - just like a sub-chart.

<img width="986" height="630" alt="image" src="https://github.com/user-attachments/assets/b1b4f4dd-03be-4a88-b51f-df67df3115a4" />